apps/studio/src/templates/layouts/AdminSidebarOnlyLayout.tsx

```diff
@@ -0,0 +1,77 @@
+import type { PropsWithChildren } from "react"
+import { useRouter } from "next/router"
+import { Flex } from "@chakra-ui/react"
+import {
+  BiCog,
+  BiFolder,
+  BiGroup,
+  BiHelpCircle,
+  BiLinkExternal,
+  BiLogOut,
+} from "react-icons/bi"
+
+import type { CmsSidebarItem } from "~/components/CmsSidebar/CmsSidebarItems"
+import { EnforceLoginStatePageWrapper } from "~/components/AuthWrappers"
+import { CmsSidebar, CmsSidebarOnlyContainer } from "~/components/CmsSidebar"
+import { useMe } from "~/features/me/api"
+import { type GetLayout } from "~/lib/types"
+
+export const AdminSidebarOnlyLayout: GetLayout = (page) => {
+  return (
+    <EnforceLoginStatePageWrapper>
+      <Flex minH="$100vh" flexDir="row" bg="base.canvas.alt" pos="relative">
+        <CmsSidebarWrapper>{page}</CmsSidebarWrapper>
+      </Flex>
+    </EnforceLoginStatePageWrapper>
+  )
+}
+
+// Extracted out since this needs to be a child of EnforceLoginStatePageWrapper
+const CmsSidebarWrapper = ({ children }: PropsWithChildren) => {
+  const router = useRouter()
+  const siteId = String(router.query.siteId)
```

Comment:
Nit: can probably use useQueryParse hook instead

Reply:
done for this and the original one